src/ScriptBuilder.Tests/Saga/SagaDefinitionReaderTest.cs

```diff
@@ -204,8 +204,9 @@ public class SagaData : ContainSagaData
 
         protected override string CorrelationPropertyName
         {
-            //Explicitly not use expression body
+#pragma warning disable IDE0025 // Use expression body for properties
```

Comment:
Should the comment be left in? It was there to indicate the intent.

Reply:
And the new comment conveys the same intent.